packages/design-system/src/components/StepList/Step.jsx

```diff
@@ -30,9 +30,24 @@ export const Step = ({ step, ...props }) => {
     'ds-c-step__content--with-content': step.description || step.steps,
   });
   const { actionsLabelText, substepsLabelText, descriptionLabelText } = props;
-  const actionsLabel = actionsLabelText.replace('%{step}', step.heading || step.title);
-  const substepsLabel = substepsLabelText.replace('%{step}', step.heading || step.title);
-  const descriptionLabel = descriptionLabelText.replace('%{step}', step.heading || step.title);
+  const actionsLabel =
+    actionsLabelText === ''
```

Comment:
What do you think about abstracting this out into a helper function? something like `getAriaLabel`

Reply:
```
const getAriaLabel = (text) => {
   const isValidTemplate = text && text.length > 0;
   const label = isValidTemplate ? text.replace('%{step}', step.heading || step.title) : undefined;
   return { 'aria-label': label };
} 
```

Reply:
That's great! Looks totally cleaning!

Reply:
I thought committing the the change with a function helper would give me Eslint error, but it didn't. Which is great, but I don't understand why I get Eslint error when I add helper methods in other react function components?